api/filrewardsd/pb/filrewards.proto

```diff
@@ -0,0 +1,101 @@
+syntax = "proto3";
+package api.filrewardsd.pb;
+option go_package = "github.com/textileio/textile/v2/api/filrewardsd/pb";
+
+import "google/protobuf/timestamp.proto";
+import "api/analyticsd/pb/analytics.proto";
+
+enum RewardType {
+  REWARD_TYPE_UNSPECIFIED = 0;
+  REWARD_TYPE_FIRST_KEY_ACCOUNT_CREATED = 1;
+  REWARD_TYPE_FIRST_KEY_USER_CREATED = 2; // does this fit "register first user?"
+  REWARD_TYPE_FIRST_ORG_CREATED = 3;
+  REWARD_TYPE_INITIAL_BILLING_SETUP = 4;
+  REWARD_TYPE_FIRST_BUCKET_CREATED = 5;
+  REWARD_TYPE_FIRST_BUCKET_ARCHIVE_CREATED = 6;
+  REWARD_TYPE_FIRST_MAILBOX_CREATED = 7;
+  REWARD_TYPE_FIRST_THREAD_DB_CREATED = 8;
+}
+
+message Reward {
+  string org_key = 1;
+  string dev_key = 2;
+  api.analyticsd.pb.AccountType account_type = 3;
+  RewardType type = 4;
+  int32 factor = 5;
+  int32 base_atto_fil_reward = 6;
+  google.protobuf.Timestamp created_at = 7;
+}
+
+message Claim {
+  string org_key = 1;
+  string claimed_by = 2;
+  int32 amount = 3;
+  google.protobuf.Timestamp created_at = 4;
+}
+
+message ProcessAnalyticsEventRequest {
+  string org_key = 1;
+  string dev_key = 2;
+  api.analyticsd.pb.Event analytics_event = 3;
+}
+
+message ProcessAnalyticsEventResponse {
+  Reward reward = 1;
+}
+
+message ListRewardsRequest {
+  string org_key_filter = 1;
+  string dev_key_filter = 2;
+  RewardType reward_type_filter = 3;
+  bool ascending = 4;
+  google.protobuf.Timestamp start_at = 5;
+  int64 limit = 6;
+}
+
+message ListRewardsResponse {
+  repeated Reward rewards = 1;
+  bool more = 2;
+  google.protobuf.Timestamp more_start_at = 3;
+}
+
+message ClaimRequest {
+  string org_key = 1;
+  string claimed_by = 2;
+  int32 amount = 3;
+}
+
+message ClaimResponse {
+}
+
+message ListClaimsRequest {
+  string org_key_filter = 1;
+  string claimed_by_filter = 2;
+  bool ascending = 3;
+  google.protobuf.Timestamp start_at = 4;
+  int64 limit = 5;
+}
+
+message ListClaimsResponse {
+  repeated Claim claims = 1;
+  bool more = 2;
+  google.protobuf.Timestamp more_start_at = 3;
+}
+
+message BalanceRequest {
+  string org_key = 1;
+}
+
+message BalanceResponse {
+  int32 rewarded = 1;
+  int32 claimed = 2;
+  int32 available = 3;
+}
+
+service FilRewardsService {
+  rpc ProcessAnalyticsEvent(ProcessAnalyticsEventRequest) returns (ProcessAnalyticsEventResponse) {}
+  rpc ListRewards(ListRewardsRequest) returns (ListRewardsResponse) {}
+  rpc Claim(ClaimRequest) returns (ClaimResponse) {}
+  rpc ListClaims(ListClaimsRequest) returns (ListClaimsResponse) {}
+  rpc Balance(BalanceRequest) returns (BalanceResponse) {}
```

Comment:
@andrewxhill in response to our convo, check out the new claim related APIs in here.

- `Balance` returns total rewarded, total claimed, and available to claim amounts for an org.
- `ListClaims` lists completed claims for an org and/or dev.
- `Claim` claims any amount of FIL for an org.

There is still some work to do here with tracking `pending` and `complete` status on claims (which will be used once we actually integrate the transferring of FIL). For the the claim just happens instantaneously and is only reflected in the filrewards database (no FIL transferred to any wallet address)